CIPs/CIP-19/CIP-19.md

```diff
@@ -0,0 +1,180 @@
+
+### Definition content
+
+**Deployment:** `<definition-DocID>`
+
+```json
+{
+  "name": "Basic Profile",
+  "description": "Basic profile information for a DID",
+  "schema": "<reference-schema-DocID>",
+  "config": {
+    "tags": ["BasicProfile"]
+  }
+}
+```
+
+### Reference Schema
+
+The Basic Profile schema defines the format of a document that contains the properties listed below. Properties not defined in the schema *cannot* be included in the Basic Profile, however the schema can always be updated via a new CIP.
+
+| Property           | Description                    | Value                                                        | Max Size | Required | Example                      |
+| ------------------ | ------------------------------ | ------------------------------------------------------------ | -------- | -------- | ---------------------------- |
+| `name`             | a name                         | string                                                       | 150 char | optional | Mary Smith                   |
+| `image`            | an image                       | IPFS CID                                                     |          | optional |                              |
+| `description`      | a short description            | string                                                       | 420 char | optional | This is my cool description. |
+| `emoji`            | an emoji                       | unicode                                                      | 2 char   | optional | 🔢                            |
+| `background`       | a background image (3:1 ratio) | IPFS CID                                                     |          | optional |                              |
+| `birthDate`        | a date of birth                | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)           | 10 char  | optional | 1990-04-24                   |
+| `url`              | a url                          | string                                                       | 240 char | optional | http://ceramic.network       |
+| `gender`           | a gender                       | string                                                       | 42 char  | optional | female                       |
+| `residenceCity`    | a city of residence            | string                                                       | 140 char | optional | Berlin                       |
+| `residenceCountry` | a country of residence         | [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) | 2 char   | optional | DE                           |
+| `nationalities`    | nationalities                  | array of [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) values | 40 char  | optional | CN                           |
+| `affiliations`     | affiliations                   | array of strings                                             | 240 char | optional | Ceramic Ecosystem Alliance   |
```

Comment:
Spec here states 240 char limit but the JSON schema limits to 140 char per item.

Reply:
I'll update the schema!